index.js

```diff
@@ -5,17 +5,20 @@
 const canvas = document.querySelector('canvas')
 const context = canvas.getContext('2d')
 
-canvas.width = innerWidth;
-canvas.height = innerHeight;
+canvas.width = innerWidth
+canvas.height =500;
 
 // Creates Player Character.
 class Player {
-    constructor(height,width,x,y,color) {
+    constructor(height,width,x,y,color,jumping,xVelocity,yVelocity) {
```

Comment:
Will the player class every change in size? If not, then consider using defined width and height attributes